internal/datacoord/clustering_compaction_job.go

```diff
@@ -31,54 +31,86 @@ type ClusteringCompactionJob struct {
 	//   trigger -> pipelining:
 	//              executing:
 	//              completed or failed or timeout
-	state          compactionTaskState
-	startTime      uint64
-	lastUpdateTime uint64
-	// todo: only store partial info in meta
-	pipeliningPlans []*datapb.CompactionPlan
-	executingPlans  []*datapb.CompactionPlan
-	completedPlans  []*datapb.CompactionPlan
-	failedPlans     []*datapb.CompactionPlan
-	timeoutPlans    []*datapb.CompactionPlan
-	analysisTaskID  UniqueID
+	state     compactionTaskState
+	startTime uint64
+	endTime   uint64
+	// should only store partial info in meta
+	compactionPlans      []*datapb.CompactionPlan
+	compactionPlanStates []compactionTaskState
+	analysisTaskID       UniqueID
 }
 
 func convertClusteringCompactionJob(info *datapb.ClusteringCompactionInfo) *ClusteringCompactionJob {
```

Comment:
rename to `convertToClusteringCompactionJob` ?

Reply:
ok